src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Symbology/UseDistanceCompositeSym/UseDistanceCompositeSym.xaml.cs

```diff
@@ -7,89 +7,99 @@
 // language governing permissions and limitations under the License.
 
+using ArcGISRuntime.Samples.Managers;
 using Esri.ArcGISRuntime.Geometry;
 using Esri.ArcGISRuntime.Mapping;
 using Esri.ArcGISRuntime.Symbology;
 using Esri.ArcGISRuntime.UI;
 using System;
 using System.Drawing;
+using System.Threading.Tasks;
 
 namespace ArcGISRuntime.WPF.Samples.UseDistanceCompositeSym
 {
     [ArcGISRuntime.Samples.Shared.Attributes.Sample(
         "Distance composite symbol",
         "Symbology",
         "This sample demonstrates how to create a `DistanceCompositeSceneSymbol` with unique marker symbols to display at various distances from the camera.",
-        "1. When the Scene initially loads, you will be looking at a point graphic symbolized with a 3D cone symbol.\n2. Zoom slightly closer to the graphic and the symbol will change to a blue 3D cube symbol.\n3. Zoom farther from the graphic and the symbol will change back to a 3D cone and then to a simple marker symbol.")]
-
+        "1. When the Scene initially loads, you will be looking at a point graphic symbolized with a 3D model symbol (airplane).\n2. Zoom away slightly from the graphic and the symbol will change to a blue 3D cone symbol (pointing in the direction of flight).\n3. Zoom farther from the graphic and the symbol will change to a blue circle marker symbol.")]
     public partial class UseDistanceCompositeSym
     {
-        // URL for an image service to use as an elevation source
-        private string _elevationSourceUrl = @"https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
-
         public UseDistanceCompositeSym()
         {
             InitializeComponent();
 
-            // Create the Scene, basemap, graphic, and composite symbol
+            // Create the Scene, basemap, graphic, and composite symbol.
             Initialize();
         }
 
-        private void Initialize()
+        private async Task Initialize()
         {
-            // Create a new Scene with an imagery basemap
+            // Create a new Scene with an imagery basemap.
             Scene myScene = new Scene(Basemap.CreateImagery());
 
-            // Create an elevation source for the Scene
-            ArcGISTiledElevationSource elevationSrc = new ArcGISTiledElevationSource(new Uri(_elevationSourceUrl));
-            myScene.BaseSurface.ElevationSources.Add(elevationSrc);
-
-            // Add the Scene to the SceneView
+            // Add the Scene to the SceneView.
             MySceneView.Scene = myScene;
 
-            // Create a new GraphicsOverlay and add it to the SceneView
+            // Create a new GraphicsOverlay and add it to the SceneView.
             GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
             graphicsOverlay.SceneProperties.SurfacePlacement = SurfacePlacement.Relative;
             MySceneView.GraphicsOverlays.Add(graphicsOverlay);
 
```

Comment:
Nathan wants to see all 'await'ed lines of code wrapped in try/catch block:

            try
            {